pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StartBrokerCommand.java

```diff
@@ -141,6 +136,13 @@ public boolean execute()
     Map<String, Object> properties = new HashMap<>();
     if (_configFileName != null) {
       properties.putAll(PinotConfigUtils.readConfigFromFile(_configFileName));
+      // Override the zkAddress and clusterName to ensure ServiceManager is connecting to the right Zookeeper and Cluster.
+      if (properties.containsKey(CommonConstants.Helix.CONFIG_OF_ZOOKEEPR_SERVER) && properties.get(CommonConstants.Helix.CONFIG_OF_ZOOKEEPR_SERVER) != null) {
+        _zkAddress = properties.get(CommonConstants.Helix.CONFIG_OF_ZOOKEEPR_SERVER).toString();
+      }
```

Comment:
Can be simplified, same for other places
```suggestion
      Object zkAddressProperty = properties.get(CommonConstants.Helix.CONFIG_OF_ZOOKEEPR_SERVER);
      if (zkAddressProperty != null) {
        _zkAddress = zkAddressProperty.toString();
      }
```

Reply:
`_zkAddress = MapUtils.getString(properties, CommonConstants.Helix.CONFIG_OF_ZOOKEEPR_SERVER, _zkAddress)` 😛

Reply:
done